usr.bin/sockstat/sockstat.1

```diff
@@ -237,6 +250,8 @@ $ sockstat -6 -P tcp
 .Xr inet 4 ,
 .Xr inet6 4 ,
 .Xr protocols 5
+.Xr libxo 3 ,
```

Comment:
We probably don't need to link libxo.3 in a section 1 man page.  I think that xo_options.7 should be sufficient.